.github/workflows/root-workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Root Workflow
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  automated-fixes: 
```

Comment:
Is there some way we can hide the `Root Workflow` prefix from here? Or just shorten it a little
![image](https://github.com/yext/studio/assets/23005393/d5bb14de-07df-463a-9182-74b1ceae71dc)
Since they all have Root Workflow on them now it's a little redundant
I noticed the SDK PR checks don't have a prefix despite having a similar flow but I can't quite figure out what's different about theirs
https://github.com/yext/answers-search-ui/pull/1832

Reply:
I've done some digging in their workflows and online and for some reason I can't find how they do it. For now I'll shorten it to `run`

Reply:
changed to `main`